templates/docs/examples/patterns/hero/hero-75-25.html

```diff
@@ -0,0 +1,38 @@
+{% extends "_layouts/examples.html" %}
+{% from "_macros/hero.jinja" import hero %}
+
+{% block title %}Hero / 75/25{% endblock %}
+{% block standalone_css %}patterns_hero{% endblock %}
+
+{% set is_paper = true %}
+{% block content %}
+{% call(slot) hero(
+  title_text='<code>H1</code> - ideally one line, up to two',
+  subtitle_text='<code>H2</code> placeholder - aim for one line, 2 is acceptable, more - use a paragraph',
+  layout='75/25',
+  is_split_on_tablet=true
+) %}
+  {#
+    Note for review: this has been set to split on tablet and hide the tall image on small sizes.
+    No Design example was provided for how this should look on medium/small, so I used best judgement.
+  #}
```

Comment:
@lyubomir-popov I couldn't find a design example of how this should look on medium/small, so I set this up to split on medium & hide the image on small (since the image is so tall). Let me know your thoughts